MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToolBar.xaml

```diff
@@ -301,8 +299,8 @@
     <Setter Property="Margin" Value="8,0,8,0" />
   </Style>
   <Style x:Key="{x:Static ToolBar.SeparatorStyleKey}" TargetType="Separator">
-    <Setter Property="Background" Value="{DynamicResource MaterialDesignDivider}" />
-    <Setter Property="BorderBrush" Value="{DynamicResource MaterialDesignDivider}" />
+    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.TextBox.HoverBackground}" />
```

Comment:
Should `.TextBox.` be a `.ToolBar.` variant here and below?

Reply:
Should be ToolBar